### ISSUE TITLE
Project is being Sunset

### ISSUE THREAD

**Issue by MikeGoldsmith** [honeycombio/honeycomb-opentelemetry-go#205]
This project has outlived its useful lifespan, and we intend to deprecate it in the future. Honeycomb recommends using the official [OpenTelemetry Go SDK](https://github.com/open-telemetry/opentelemetry-go), and using individual contrib components as required. The official OpenTelemetry Go SDK is stable and has grown to offer most of the ergonomic benefits that this distribution has historically offered.

If you are actively using this distribution and feel the official OpenTelemetry SDK does not meet your needs, please file an issue here with your use case.

The following links may be useful:

* [Honeycomb OSS lifecycle and practices.](https://github.com/honeycombio/home/blob/main/honeycomb-oss-lifecycle-and-practices.md)
* Configuring the [OpenTelemetry SDK for language](https://docs.honeycomb.io/send-data/go/opentelemetry-sdk/) with Honeycomb.

**Comment by nordfjord** [comment]
> If you are actively using this distribution and feel the official OpenTelemetry SDK does not meet your needs

We're using the distribution, but the main thing we use the Honeycomb distro for is the BaggageSpanProcessor, will that be released as a separate (maintained) package, or is there no need for it anymore to handle multi-span attributes?

**Comment by josegonzalez** [comment]
Should this be called out in the docs here? https://docs.honeycomb.io/send-data/go/honeycomb-distribution/

**Comment by MikeGoldsmith** [comment]
> > If you are actively using this distribution and feel the official OpenTelemetry SDK does not meet your needs
> 
> We're using the distribution, but the main thing we use the Honeycomb distro for is the BaggageSpanProcessor, will that be released as a separate (maintained) package, or is there no need for it anymore to handle multi-span attributes?

Yes, we're in the process of contributing the baggage span processor to [otel-go-contrib](https://github.com/open-telemetry/opentelemetry-go-contrib/pull/5404) so all OpenTelemetry users can benefit from multi-span attributes.

This project will continue to be maintained until we're confident it's no longer required but also want to indicate our intentions and gather feedback.

**Comment by MikeGoldsmith** [comment]
> Should this be called out in the docs here? https://docs.honeycomb.io/send-data/go/honeycomb-distribution/

Agreed - we will update documentation when we're ready to take steps towards sunsetting this project. For now, we intend to continue to maintain this project and are gathering feedback from users.

**Comment by MikeGoldsmith** [comment]
No objections, issues, or non-dependabot PRs have been submitted since this sunsetting issue was posted. If we receive no further objections, we will archive the repo on August 12, 2025.

**Comment by codeboten** [comment]
No further objections received, we will now archive the repo.
